oojs/application.js

```diff
@@ -1,13 +1,30 @@
 $(document).ready(function() {
+  function die() {
```

Comment:
You can move this code and the other two functions out of the `$(document).ready()` block so that it doesn't need to wait for the DOM to render.